data/db.js

```diff
@@ -0,0 +1,36 @@
+const knex = require('knex');
+const knexfile = require('../knexfile');
+
+const db = knex(knexfile.development);
+
+module.exports = {
+    
+    getProject: id => {
+        if (id) {
+            return db.select().from('projects').where({ id });
+        } else {
+            return db.select().from('projects');
+        }
+    },
+
+    addProject: project => {
+        return db.insert(project).into('projects');
+    },
+
+    getProjectActions: id => {
+        return db.select().from('actions').where('project_id', id);
+    },
+
+    getActions: id => {
+        if (id) {
+            return db.select().from('actions').where({ id });
+        } else {
+            return db.select().from('actions');
+        }
+    },
+
+    addAction: action => {
+        return db.insert(action).into('actions');
+    }
```

Comment:
Your get and add for projects and actions look identical. Why not break them out into helper funcs? For example:
```javascript
// ...

const get = table => (id = NaN) => {
  if (isNaN(id)) {
    return db(table);
  }

  return db(table).where({ id });
};

const add = table => (
  data => db.insert(data).into(table)
);

module.exports = {
  getProjects: get('projects'),

  addProject: add('projects'),

  getProjectActions: id => (
    db('actions').where('project_id', id)
  ),

  getActions: get('actions'),

  addAction: add('actions')
};
```